### PR TITLE
[Boids] Fix cohesion bug

### PR DESCRIPTION
**Pull request by tristanbob**
Thanks to @planktonfun for the fix!

### CODE REVIEW

File: extensions/reviewed/BoidsMovement.json [hunk 1]
```diff
@@ -2118,6 +2124,50 @@
             }
           ],
           "objectGroups": []
+        },
+        {
+          "description": "Get Boids Priority",
```
Comment (tristanbob) [inline extensions/reviewed/BoidsMovement.json:2129]: What does this do?

File: extensions/reviewed/BoidsMovement.json [hunk 2]
```diff
@@ -387,6 +389,9 @@
                 "      sumY += otherObject.getCenterYInScene();",
                 "      count++;",
                 "    }",
+                "",
+                "    this.nearby = count;",
```
Comment (tristanbob) [inline extensions/reviewed/BoidsMovement.json:393]: I don't see anything using this value.  What is it used for?

Comment (planktonfun) [reply extensions/reviewed/BoidsMovement.json:393]: Oops I used it for debugging

File: extensions/reviewed/BoidsMovement.json (hunk 1 above)
Comment (planktonfun) [reply extensions/reviewed/BoidsMovement.json:2129]: Gets the current action of the boid

**Comment by D8H** [comment]
I fixed a scene change issue that seems to solve the issue of the paper-rock-scissors example:
- https://github.com/GDevelopApp/GDevelop-extensions/pull/704

Let me know if the changes of this PR fix something else.

**Comment by tristanbob** [comment]
This cohesion bug was fixed by #704.  I'm closing this PR.

**Pull request closed without merging**